packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -193,6 +194,21 @@ export const ComponentShowcase = () => {
 
   return (
     <div className="ComponentShowcase">
+      <CWText type="h3">Multi select list</CWText>
+      <SelectList
+        placeholder="Add or select a chain"
+        isMulti
+        isClearable={false}
+        defaultValue={[{ value: 'solana', label: 'Solana' }]}
+        options={[
+          { value: 'solana', label: 'Solana' },
+          { value: 'polkadot', label: 'Polkadot' },
+          { value: 'ethereum', label: 'Ethereum' },
+          { value: 'substrate', label: 'Substrate' },
+          { value: 'binance', label: 'Binance' },
+        ]}
+      />
+      <div style={{ height: '500px' }} />
```

Comment:
this div is not necessary

Reply:
Yup, this was just to give space for the dropdown (again its a sample page right?)

Reply:
Removed